extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -18,8 +19,9 @@ const makeDisplaySet = instances => {
     value: isReconstructable,
     averageSpacingBetweenFrames,
   } = isDisplaySetReconstructable(instances);
-
   // set appropriate attributes to image set...
+  const messages = validateInstances(instances, isReconstructable);
```

Comment:
it is not validating (no error is thrown as validation implies) it is more `getDisplaySetMessages` or similar